foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -908,13 +775,14 @@ class StringCallback : Java.Lang.Object, IAdobeCallback
 
 ## getIdentifiers
 
+This API returns all customer identifiers that were previously synced with the Adobe Experience Cloud.
```

Comment:
... synced with the ECID Service

Reply:
Should I use "Adobe Experience Cloud ID Service" instead?

Reply:
Hmm. Or is it "Adobe Experience Cloud Identity Service"? Then would repeated mentions be "Identity Service" or "ID Service" or just "ECID"? In the Experience Cloud release notes, it seems "ECID" is used interchangeably with the Identity service as a whole and with the Experience Cloud ID.
https://experienceleague.adobe.com/docs/id-service/using/release-notes/release-notes.html?lang=en